src/main.rs

```diff
@@ -116,21 +117,49 @@ pub struct PowChallenge {
     difficulty: u8,
 }
 
+/// Chain: L1 for Bitcoin. L2 for Strata.
```

Comment:
The "L1 for Bitcoin. L2 for Strata." should be docstrings on the variants not on the enum.

What about:
```suggestion
/// Which chain the faucet is reasoning about.
```